root/static/scripts/relationship-editor/release.js

```diff
@@ -206,7 +203,11 @@ const RE = MB.relationshipEditor = MB.relationshipEditor || {};
             }));
 
             var trackCount = _.reduce(release.mediums(),
-                function (memo, medium) { return memo + medium.tracks.length }, 0);
+                function (memo, medium) {
+                    return memo + medium.tracks.length;
+                },
+                0
```

Comment:
I'd also move release.mediums() to its own line, and you'd want a comma-dangle after 0 while you're at it.